GooglePlayInstant/Editor/Bundletool.cs

```diff
@@ -0,0 +1,105 @@
+// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System.IO;
+using GooglePlayInstant.Editor.GooglePlayServices;
+using UnityEngine;
+
+namespace GooglePlayInstant.Editor
+{
+    /// <summary>
+    /// Provides methods for <a href="https://developer.android.com/studio/command-line/bundletool">bundletool</a>.
+    /// </summary>
+    public static class Bundletool
+    {
+        public const string BundletoolVersion = "0.6.0";
+
+        /// <summary>
+        /// BundleTool config optimized for Unity-based instant apps.
+        /// Config definition: https://github.com/google/bundletool/blob/master/src/main/proto/config.proto
+        /// SplitsConfig:
+        ///  - Split on ABI so only one set of native libraries (armeabi-v7a, arm64-v8a, or x86) is sent to a device.
+        ///  - Do not split on LANGUAGE since Unity games don't store localized strings in the typical Android manner.
+        ///  - Do not split on SCREEN_DENSITY since Unity games don't have per-density resources other than app icons.
+        /// UncompressNativeLibraries: Instant apps have smaller over-the-wire and on-disk size with this enabled.
+        /// TODO: Consider fully uncompressed, i.e. ""compression"": { ""uncompressedGlob"": [""**/*""] }
+        /// </summary>
+        private const string BundleConfigJsonText = @"
+{
+  ""optimizations"": {
+        ""splitsConfig"": {
+            ""splitDimension"": [
+                {
+                    ""value"": ""ABI"",
+                    ""negate"": false
+                },
+                {
+                    ""value"": ""LANGUAGE"",
+                    ""negate"": true
+                },
+                {
+                    ""value"": ""SCREEN_DENSITY"",
+                    ""negate"": true
+                }
+            ]
+        },
+        ""uncompressNativeLibraries"": { ""enabled"": true }
+    }
+}";
```

Comment:
Should the section above be indented?

Reply:
Discussed offline. Since this is a multi-line verbatim string, the consideration was that any indentation here would be carried over to the file that is written out. We agreed to leave this as-is.